ts/screens/authentication/SpidCIEInformationScreen.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * A screen where the user can know more about SPID, CIE and access to spid.gov.it
+ */
+import {
+  Button,
+  Col,
+  Container,
+  Content,
+  Grid,
+  H2,
+  Row,
+  Tab,
+  Tabs,
+  Text,
+  Toast,
+  View
+} from "native-base";
+import * as React from "react";
+import { Linking } from "react-native";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import Markdown from "../../components/ui/Markdown";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+import variables from "../../theme/variables";
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>;
+};
+
+type Props = OwnProps;
+
+const styles = StyleSheet.create({
+  value: {
+    alignSelf: "flex-start",
+    color: variables.brandPrimary,
+    paddingTop: 2
+  },
+  row: {
+    flexDirection: "row",
+    alignItems: "center"
+  },
+  tabBarContainer: {
+    elevation: 0
+  },
+  tabBarUnderline: {
+    borderBottomColor: customVariables.brandPrimary,
+    borderBottomWidth: customVariables.tabUnderlineHeight
+  }
+});
+
+class SpidCIEInformationScreen extends React.Component<Props, never> {
+  private getValueContent(value: string, content: string) {
+    return (
+      <Row style={styles.row}>
+        <Col size={1}>
+          <H2 style={styles.value}>{value}</H2>
+        </Col>
+        <Col size={5}>
+          <Text>{content}</Text>
+        </Col>
+        <Col size={2} />
+      </Row>
+    );
+  }
+
+  private browseToLink(url: string) {
+    Linking.openURL(url).catch(() => {
+      Toast.show({ text: I18n.t("genericError") });
+    });
+  }
+
+  public render() {
+    return (
+      <BaseScreenComponent goBack={true}>
+        <Container>
+          <View style={{ margin: 24 }}>
```

Comment:
I suggest here to insert it into styles as `{padding: customVariables.contentPadding}`